version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.0.2-SNAPSHOT"
+version in ThisBuild := "1.2.0-SNAPSHOT"
```

Comment:
Is there a reason this is incremented to `1.2.0` instead of `1.1.0`?

Reply:
Nope, that was a mistake. Addressed in 0581e11.